packages/governance/test/swingsetTests/committeeBinary/vat-voter.js

```diff
@@ -43,7 +43,7 @@ const build = async (log, zoe) => {
         },
       });
       const subscription = E(electoratePublicFacet).getQuestionSubscription();
-      observeIteration(subscription, votingObserver);
+      await observeIteration(subscription, votingObserver);
```

Comment:
Similarly here. Isn't `await` misleading? The point of the code is to set the observer off on it's own, and return the facet. Why is `await` better?

Reply:
Attempting to channel @erights: I expect some callers to want to be able to react to the results. In this test, it seems that the test should be conditional on results of the observation, so it doesn't really need to detect a failure in the adapter.